ui/src/app/components/help-sidebar.tsx

```diff
@@ -80,6 +81,26 @@ const styles = reactStyles({
   contentItem: {
     marginTop: 0,
     color: colors.primary
+  },
+  textSearch: {
+    width: '100%',
+    borderRadius: '4px',
+    backgroundColor: colorWithWhiteness(colors.primary, .95),
+    marginTop: '5px',
+    color: colorWithWhiteness(colors.primary, -1),
```

Comment:
In general, I would want to avoid negative numbers in CSS.

Reply:
Yeah, the [Invision screen](https://projects.invisionapp.com/d/main/#/console/12820961/356269832/inspect) has this text as black.

@lstrano-broad Is this the correct color for the search bar text?
<img width="380" alt="Screen Shot 2019-09-16 at 10 22 22 AM" src="https://user-images.githubusercontent.com/40036095/64970701-f41aad00-d86b-11e9-9289-4a33704a92a0.png">

Reply:
In this case, the negative number is an input the function, which takes a color and a number from `-1` to `1` and transforms it from a scale from black to white, so that's within the scope of the function.

Reply:
Ah, I see the comment on the function now, thanks for the clarification.

Reply:
Updated to `colors.primary` per Lou's comment